project/Team.py

```diff
@@ -1,173 +1,532 @@
 import unittest
+import datetime
+import time
 from abc import ABC, abstractmethod
-from GameMaker import UserABC
 
+from Landmark import LandmarkFactory
+from Team import TeamFactory
+
+
+class GameInterface(ABC):
+    @abstractmethod
+    def add_team(self, name, password):
+        pass
+
+    @abstractmethod
+    def remove_team(self, name):
+        pass
+
+    @abstractmethod
+    def modify_team(self, oldname, newname=None, newpassword=None):
+        pass
+
+    @abstractmethod
+    def add_landmark(self, location, clue, answer):
+        pass
+
+    @abstractmethod
+    def remove_landmark(self, landmark):
+        pass
 
-class TeamI(ABC):
     @abstractmethod
-    def login(self, password):
+    def modify_landmark(self, oldlandmark, newlandmark):
         pass
 
     @abstractmethod
-    def changeName(self, name):
+    def set_point_penalty(self, points):
         pass
 
     @abstractmethod
-    def changePassword(self, password):
+    def set_time_penalty(self, time):
         pass
 
     @abstractmethod
-    def answer_question(self, answer):
+    def start(self):
         pass
 
     @abstractmethod
-    def get_status(self):
+    def end(self):
         pass
 
     @abstractmethod
-    def get_username(self):
+    def get_status(self,team):
         pass
 
     @abstractmethod
-    def get_points(self):
+    def answer_question(self,team,answer):
         pass
 
     @abstractmethod
-    def add_points(self, points):
+    def quit_question(self,team,password):
         pass
 
 
-class TeamFactory:
```

Comment:
No more Team in Team.py???? I think you had some bad copy/paste or something.

Reply:
Yeah that's what I am thinking

Reply:
Check out MAD_13.1,  I had some weird issues going on when trying to push and pull